### PR TITLE
Update probe period

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -247,7 +247,7 @@ endif
 .PHONY: catalog-build
 ## Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool $(CONTAINER_TOOL) --mode replaces --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	sudo $(OPM) index add --container-tool $(CONTAINER_TOOL) --mode replaces --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
```
Comment (pruivo) [inline Makefile:250]: sudo?

File: controllers/pods.go [hunk 2]
```diff
@@ -36,16 +36,16 @@ func PodPortsWithXsite(i *infinispanv1.Infinispan) []corev1.ContainerPort {
...
 func PodReadinessProbe() *corev1.Probe {
-	return probe(5, 10, 10, 1, 80)
+	return probe(5, 0, 10, 1, 80)
 }
 
 func PodStartupProbe() *corev1.Probe {
-	// Maximum 10 minutes (60 * 10s) to finish startup
-	return probe(60, 10, 10, 1, 80)
+	// Maximum 10 minutes (600 * 1s) to finish startup
+	return probe(600, 10, 1, 1, 80)
```
Comment (pruivo) [inline controllers/pods.go:48]: we discussed the initial delay to be 1 second too.

File: scripts/ci/install-catalog-source.sh [hunk 3]
```diff
@@ -14,7 +14,7 @@ BUNDLE_IMG_NAME=infinispan-operator-bundle
 export IMG=${IMG_REGISTRY}/infinispan-operator
 export BUNDLE_IMG=${IMG_REGISTRY}/${BUNDLE_IMG_NAME}:v${VERSION}
 export CATALOG_IMG=${IMG_REGISTRY}/infinispan-test-catalog
-export CATALOG_BASE_IMG=${CATALOG_BASE_IMG-"quay.io/operatorhubio/catalog_tmp"}
+export CATALOG_BASE_IMG=${CATALOG_BASE_IMG-"quay.io/operatorhubio/catalog:tmp_latest_sql"}
```
Comment (pruivo) [inline scripts/ci/install-catalog-source.sh:17]: don't know what this is but I'll trust you :)

File: Makefile (hunk 1 above)
Comment (ryanemerson) [reply Makefile:250]: Merged as part of https://github.com/infinispan/infinispan-operator/pull/1382.

Unfortunately `sudo` is now required with their multi-arch images due to the way they have set the permissions. Previously we were using a single-arch tag, but that's no longer available so we have to work around this way.

File: scripts/ci/install-catalog-source.sh (hunk 3 above)
Comment (ryanemerson) [reply scripts/ci/install-catalog-source.sh:17]: That's part of https://github.com/infinispan/infinispan-operator/pull/1382 as well.

File: controllers/pods.go (hunk 2 above)
Comment (ryanemerson) [reply controllers/pods.go]: Whoops, good catch!

**Comment by ryanemerson** [comment]
@pruivo Do we want to reconsider the GossipRouter probe values as well?

**Comment by pruivo** [comment]
> @pruivo Do we want to reconsider the GossipRouter probe values as well?

probably :) 
the startup could be 1 sec initial delay and 1 sec period (I believe the gossip router starts in under 10 ms)
the liveness/readiness can have their initial delay to 0.

**Comment by ryanemerson** [comment]
Updated

**Comment by ryanemerson** [comment]
CI looks good :slightly_smiling_face:

**Comment by pruivo** [comment]
merged! thanks @ryanemerson !

**Pull request merged**